tools/src/org.graalvm.tools.lsp.test/src/org/graalvm/tools/lsp/instrument/TestInstrument.java

```diff
@@ -0,0 +1,40 @@
+package org.graalvm.tools.lsp.instrument;
+
+import org.graalvm.options.OptionCategory;
+import org.graalvm.options.OptionDescriptors;
+import org.graalvm.options.OptionKey;
+import org.graalvm.tools.lsp.api.ContextAwareExecutorRegistry;
+import org.graalvm.tools.lsp.api.VirtualLanguageServerFileProvider;
+import org.graalvm.tools.lsp.instrument.TestInstrumentOptionDescriptors;
+import org.graalvm.tools.lsp.server.TruffleAdapter;
+
+import com.oracle.truffle.api.instrumentation.TruffleInstrument;
+import com.oracle.truffle.api.instrumentation.TruffleInstrument.Registration;
+
+@Registration(id = "lspTestInstrument", name = "LspTestInstrument", version = "0.1", services = {VirtualLanguageServerFileProvider.class, ContextAwareExecutorRegistry.class,
+                TruffleAdapterProvider.class})
```

Comment:
Shouldn't it be `internal = true`? If I got it right, `<language launcher> --lsp` does nothing, right? The LSP server should be started via `GraalLanguageServerLauncher` only, right?